### PR TITLE
[1.26] pin virtualenv to <20.22.0 for python-version==2.7

### PR DESCRIPTION
**Pull request by graingert**
backport of https://github.com/urllib3/urllib3/pull/2977

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -94,11 +94,16 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - name: Set Up Python 3 to run nox
-        if: matrix.python-version != '3.7'
+        if: matrix.python-version != '2.7'
```
Comment (graingert) [inline .github/workflows/ci.yml:97]:
```suggestion
        if: matrix.python-version == '2.7'
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -94,11 +94,16 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - name: Set Up Python 3 to run nox
-        if: matrix.python-version != '3.7'
+        if: matrix.python-version == '2.7'
```
Comment (graingert) [inline .github/workflows/ci.yml:97]: this is a bit of a drive by change I can move it into a separate PR

**Comment by pquentin** [comment]
Thanks, this will buy us much needed time. Any idea why that fails with "every step must define a `uses` or `run` key"?

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -94,11 +94,16 @@ jobs:
...
         uses: actions/setup-python@v4
         with:
           python-version: "3"
 
+      - if: matrix.python-version == '2.7'
+        name: Install Dependencies
+        run: python -m pip install --upgrade pip setuptools nox 'virtualenv<20.22.0'
+
+      - if: matrix.python-version != '2.7'
       - name: Install Dependencies
```
Comment (pquentin) [inline .github/workflows/ci.yml:107]: Aha! Found it

```suggestion
        name: Install Dependencies
```

**Review by sethmlarson — APPROVED** [review]
LGTM, thank you!

**Pull request merged**